src/core/sock/sockinfo_tcp.cpp

```diff
@@ -5692,18 +5692,24 @@ void sockinfo_tcp::tcp_tx_pbuf_free(void *p_conn, struct pbuf *p_buff)
     sockinfo_tcp *p_si_tcp = (sockinfo_tcp *)(((struct tcp_pcb *)p_conn)->my_container);
     dst_entry_tcp *p_dst = (dst_entry_tcp *)(p_si_tcp->m_p_connected_dst_entry);
 
```

Comment:
I thought of 2 alternatives here:
1. a global static - with eager initialization - doable.
2. a member in the class - enlarging the already large instance - didn't like it - but doable.
3. lazy initialization with a local static - felt more readable (yet debateable) - yet you do pay a small perf price on first call  to tcp_tx_pbuf_free